src/RA_StringUtils.h

```diff
@@ -45,6 +45,7 @@ _NODISCARD inline auto StringPrintf(_In_z_ _Printf_format_string_ const CharT* c
     }
     else if constexpr(std::is_same_v<CharT, wchar_t>)
     {
+        sFormatted.reserve(512); // The number is arbitrary but the default capacity is too small (7)
```

Comment:
Had to explicitly reserve more memory for `std::wstring`, or the return would be `-1` for `std::vswprintf`.